docs/minimal_example.rst

```diff
@@ -144,5 +144,11 @@ Here's the HTML for the cell above:
    plt.plot(x, np.sin(x))
    plt.plot(x, np.cos(x))
    </pre>
+   
+And here's an example where the contents cannot be modified once instantiated:
```

Comment:
Hey all - sorry I didn't get to this earlier. I think it's fine to merge but we really need to document things better than this if we add new features.

Also @moorepants could you please check out [the executablebooks guidelines for merge commits](https://executablebooks.org/en/latest/contributing.html#commit-messages) and format the squash-and-merge message accordingly? Thanks!